docs/socketexample.rst

```diff
@@ -552,7 +569,7 @@ need this method.
 .. doctest::
 
    >>> utils = list(zope.component.getAllUtilitiesRegisteredFor(IUSSocket))
-   >>> utils.sort()
+   >>> utils.sort(key=lambda x: type(x).__name__)
```

Comment:
Prefer `sorted()` to creating a list and then sorting it in place (which I see you've done elsewhere).